src/components/Header.js

```diff
@@ -311,7 +311,7 @@ export const mapStateToProps = state => {
       !languages.loading &&
       !code.parsing &&
       (validServerUrl || !options.customServer) &&
-      code.dirty,
```

Comment:
Why `canParse` will be true even if `!code.dirty`.
With that validation, it would be avoided parsing again and again once the current code has been successfully parsed.

Reply:
1) I don't see a reason why we should block re-parsing
2) But I find it useful to be able to reparse (for example I'm debugging driver, so I don't change input, I change how parsing works).

Reply:
good reasoning